playwright/package.json

```diff
@@ -5,12 +5,13 @@
   "private": true,
   "repository": "dequelabs/axe-core-maven-html",
   "scripts": {
-    "start": "http-server ./src/test/resources/fixtures -p 1337"
+    "start": "http-server ./node_modules/axe-test-fixtures/fixtures -p 1337"
```

Comment:
I cannot remember why I put all of the text-fixtures in the playwright `resources/fixtures` that was not intended. Now we pull from `node_modules` instead.